doc/source/plugins/b506_yaml_load.rst

```diff
@@ -1,5 +1,5 @@
 ---------------
```

Comment:
The file name also needs to be renamed to b612_yaml_load.rst

Reply:
Actually we need to have both.  Keep b506 for historical reference.  Similar to b111 which was also removed.  And add the new b612 as a separate file.